rebasehelper/logger.py

```diff
@@ -155,6 +155,10 @@ class ColorizingStreamHandler(logging.StreamHandler):
     def choose_color_map(self):
         self.terminal_background = rebasehelper.utils.ConsoleHelper.detect_background()
 
+    def set_color_map(self, color_map):
+        color_map = self.terminal_background if color_map == 'auto' else color_map
```

Comment:
This is not such a big deal, but it would be better if no autodetection was performed in case `color_map` is not auto. It could prevent failures in case something unexpected happens during autodetection.